sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1577,6 +1578,17 @@ def std(self, *args, **kwargs):
     # Compute variance (deferred scalar) with same args, then sqrt it
     return self.var(*args, **kwargs).apply(lambda var: math.sqrt(var))
 
+  @frame_base.with_docs_from(pd.Series)
+  @frame_base.args_to_kwargs(pd.Series)
+  @frame_base.populate_defaults(pd.Series)
+  def mean(self, skipna, **kwargs):
```

Comment:
Can we make `skipna` as keyword argument?
```suggestion
  def mean(self, skipna=True, **kwargs):
```

Reply:
Following this https://pandas.pydata.org/docs/reference/api/pandas.Series.mean.html#pandas.Series.mean

Reply:
That's actually what the decorators do - populate_defaults pulls the default values from the panda implementation of the function, so we don't have to duplicate them here.